src/client/components/DetailsModal/Details/components/DetailInput.tsx

```diff
@@ -155,6 +156,45 @@ export const AreaInput: React.FC<AreaInputProps> = ({
   />
 )
 
+type MileageInputProps = {
+  name: string
+  label: string
+  formikProps: FormikProps<QuoteInput>
+}
+
+export const MileageInput: React.FC<MileageInputProps> = ({
+  name,
+  label,
+  formikProps,
+}) => {
+  const { isoLocale } = useCurrentLocale()
+  const textKeys = useTextKeys()
+  const unitMultipler = isoLocale.toLowerCase().includes('en') ? 10 : 1
```

Comment:
This isn't always true. Not sure what it should be if we would ever launch it outside of Sweden, but let's not assume that anything non-English will be a factor 1. Better to use a translation like I suggested below.